great_expectations/cli/project.py

```diff
@@ -61,7 +61,18 @@ def project_upgrade(directory):
 
 def do_config_check(target_directory):
     try:
-        context = DataContext(context_root_dir=target_directory)
+        context: DataContext = DataContext(context_root_dir=target_directory)
+        ge_config_version: int = context.get_config().config_version
+        if int(ge_config_version) < CURRENT_GE_CONFIG_VERSION:
+            upgrade_message: str = f"""The config_version of your great_expectations.yml -- {float(ge_config_version)} -- is outdated.
+Please consult the 0.13.x migration guide ttps://docs.greatexpectations.io/en/latest/guides/how_to_guides/migrating_versions.html and
+upgrade your Great Expectations configuration to version {float(CURRENT_GE_CONFIG_VERSION)} in order to take advantage of the latest capabilities.
+            """
+            return (
+                False,
+                upgrade_message,
+                None,
+            )
```

Comment:
What happened to routing users through `load_data_context_with_error_handling` so they have opportunity to upgrade using cli tool at same time?

Reply:
@roblim This does not work for `check-config` -- it has to investigate the current state and only report on the potential upgrade opportunity.  If it offered an upgrade right there, it would not be able to print the message.

Reply:
Gotcha. Okay.